crates/benchmark-runner/src/guest_programs.rs

```diff
@@ -64,29 +64,43 @@ pub fn stateless_validator_generate_inputs(
     Ok(guest_inputs)
 }
 
-/// Metadata for the block RLP length calculation guest program.
+/// Metadata for the block block length calculation guest program.
 #[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct RLPLengthMetadata {
+pub struct BlockEncodingLengthMetadata {
+    format: String,
```

Comment:
Since the guest program was generalized, now the encoding format is part of the metadata to know by looking at the results which format was used for the run and potential comparisons.